src/makemsix/main.cpp

```diff
@@ -567,6 +659,7 @@ int main(int argc, char* argv[])
         #ifdef MSIX_PACK
         CreatePackCommand(),
         #endif
+        CreateBundleCommand(),
```

Comment:
>CreateBundleCommand(), [](start = 8, length = 22)

This should only be present if MSIX_PACK

Reply:
Updated.